ph5/utilities/ph5validate.py

```diff
@@ -847,14 +831,32 @@ def get_args():
 
     args = parser.parse_args()
 
-    # Add formatter
-    args.level = str(args.level).upper()
-    if args.level != "ERROR" and args.level != "WARNING" and \
-            args.level != "INFO":
-        raise ValueError("Invalid logging level.")
+    outfile = 'ph5_validate.log' if args.outfile is None else args.outfile
```

Comment:
this is already the default
add choices to `parser.add_argument(` above.

Reply:
I will change this to:
`outfile = args.outfile`

Reply:
> this is already the default
> add choices to `parser.add_argument(` above.

Reply:
Looks like you missed this.

Reply:
How do you think about adding help for option -o above?

```
    parser.add_argument(
        "-o", "--outfile", action="store",
        default="ph5_validate.log", type=str, metavar="outfile",
        help=("To record logging detail according to level stated in option "
              "-l. Default is ph5_validate.log"))
```

Reply:
```    parser.add_argument(
        "-l", "--level", action="store", default="WARNING",
        type=str, metavar="level", choices=('error', 'debug....
        help=("Level of logging detail. Choose from ERROR, WARNING, or INFO"))
```

Reply:
Thank you for your help. I misunderstood your comment. I thought you talked about argument -o instead of argument -l. I fixed it according to your recommendation.